src/test/resources/integrationtests/sso/testSSOFieldLauncherService.feature

```diff
@@ -0,0 +1,42 @@
+#Author: andrew.johnys@ext.ons.gov.uk
+#Keywords Summary : CCS CENSUS COVERAGE SURVEY
+#Feature: Test SSO to EQ using Field Launcher Service
+#Scenario: Authentication of SSO
+#Scenario: Display message to the field officer when the response has already submitted
+#Scenario: Field Officer is already authenticated
+#Scenario: Display message to the field officer when the case id is invalid
+#
+#The following scenario is also covered by the scenario 'Authentication of SSO'
+#Scenario: Display SSO Authentication UI
+## (Comments)
+Feature: Test SSO to EQ using Field Launcher Service
+  I want to verify that I can access the EQ using SSO for the Field Launcher Service
+
+  @SetUpFieldServiceTests @TearDown
+  Scenario: Authentication of SSO
+    Given I am a field officer and I have access to a device with SSO
+    And I click on the job URL⁠ in the chrome browser
+    And a field proxy authentication UI is displayed on the screen
+    When I enter my correct SSO credentials and click OK
+    Then the EQ launch event is triggered
+
+  @SetUpFieldServiceTests @TearDown
+  Scenario: Display message to the field officer when the response has already submitted
+    Given that the response to a CCS interview job has already been submitted
+    And I click on the job URL⁠ in the chrome browser
+    When I enter my correct SSO credentials and click OK
+    Then the completion message "The CCS Questionnaire has been completed." is displayed to me
+
+  @SetUpFieldServiceTests @TearDownMultiWindows
+  Scenario: Field Officer is already authenticated
+    Given I have already entered my credentials for SSO
```

Comment:
I can now see what this step is doing but I still think it's confusing.

Why not just replace it with the click-on-job-url and enter-sso-credentials steps? It would make it more obvious that firstly the sign in is happening and secondly that the 2nd click-on-job does not require a sign in. It would also make the code base a little smaller because it gets rid of the already-entered-my-credentials step.